### PR TITLE
N15: Add Worship The Sun

### PR DESCRIPTION
**Pull request by tuzz**
Part of #320 ("Newsletter 15")

### CODE REVIEW

File: content/posts/newsletter-015/index.md [hunk 1]
```diff
@@ -60,6 +60,37 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### Worship The Sun
+
+![Worship The Sun](worship_the_sun.jpg)
+_▲ One of the many unique and confounding puzzles in the game_
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-015/index.md:66]: Not sure if the arrow is necessary here, as the website applies formatting to the image captions to make it more obvious they're connected to the image.

File: content/posts/newsletter-015/index.md [hunk 2]
```diff
@@ -60,6 +60,37 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+and painstakingly animated.
+
+The game is a few months into development with a release target of late 2021.
+You can read about how swimming was added to the game in [GameDev Note 1:
+Taking the Plunge][plunge] which contains a sneak peak at some of the levels.
+For updates and possible playtesting opportunities, follow
+[@chrispatuzzo][chrispatuzzo].
+
+_Discussions:
+[/r/WorshipTheSunGame](https://reddit.com/r/WorshipTheSunGame)
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-015/index.md:86]: This is missing a `_` to close off the italics:
```suggestion
[/r/WorshipTheSunGame](https://reddit.com/r/WorshipTheSunGame)_
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Content looks good, just a couple of things to tweak.

**Comment by tuzz** [comment]
Thanks @17cupsofcoffee, both those things are now fixed.

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
:rocket: (made a few small style/fmt tweaks for consistency with the rest of the newsletter)

**Comment by ozkriff** [comment]
Thanks for the PR! Good luck with the game :)

**Pull request merged**